subprojects/core-api/src/main/java/org/gradle/api/invocation/Gradle.java

```diff
@@ -45,6 +45,14 @@
  */
 @HasInternalProtocol
 public interface Gradle extends PluginAware, ExtensionAware {
+
+    /**
+     * Get the path of this build relative to the root build.
+     *
+     * @since 9.1.0
+     */
+    String getBuildPath();
```

Comment:
I also considered `getPath`, but given we named this `getBuildPath` on `BuildIdentifier`, I felt this was more appropriate.

Reply:
I like the name, but I question whether it isn't better to expose this as `Path` object?

Reply:
We have had conversations about this in the past. I remember @big-guy has opinions.

Last I remember, we don't actually want users deriving hierarchy from this path, but want to allow them to use it as an identifier -- meaning exposing it as a string.

This is similar to the existing `getPath` and `getBuildTreePath` methods on `Project` which also return paths as a `String`

Reply:
I tried looking at some git/PR history but could not find a prior conversation